datastacks/datastacks/datastacks_cli.py

```diff
@@ -11,11 +11,17 @@
 @click.group()
 @click.option("--log-level", "-l", type=LogLevel(), default=logging.INFO)
 def cli(log_level):
+    """
+    Specify log level for Datastacks
+    """
```

Comment:
```suggestion
    """Specify log level for Datastacks."""
```

And the same in other places. I like this concise format of the first short line in the docstrings, e.g.: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
done